uber/config.py

```diff
@@ -646,6 +646,19 @@ def MENU_FILTERED_BY_ACCESS_LEVELS(self):
     def ADMIN_ACCESS_SET(self):
         return uber.models.AdminAccount.access_set()
 
+    @cached_property
+    def ADMIN_PAGES(self):
+        # Build a list of all site sections and their pages
+        public_site_sections = ['preregistration', 'static_views', 'landing', 'panel_applications', 'mits_applications',
```

Comment:
We'll want to eventually make this a bit more robust, particularly for pages _within_ site sections (e.g., the `login` page in accounts is obviously accessible to everyone). However, not getting around to that merely prints a few extraneous options on the access groups page, so I am 100% planning to punt that to future-me.